demo/src/test/java/com/example/test/pages/KBBMOOCProject/ButtonReadmore.java

```diff
@@ -0,0 +1,17 @@
+package com.example.test.pages.KBBMOOCProject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ButtonReadmore extends KbbBasePage {
+
+    public ButtonReadmore(WebDriver driver) {
+        super(driver);
+        //TODO Auto-generated constructor stub
+    }
+    WebElement Readmore = keyword.findWebElementByXpath("//p[text()='Read More']");
```

Comment:
should write the web element locator in separate line then call into a method/function